classes/restaurants.py

```diff
@@ -61,17 +61,20 @@ def select_restaurant_by_id(self, r_id):
             query = """SELECT * FROM RESTAURANTS WHERE id = %s"""
             cursor.execute(query, [r_id])
             value = cursor.fetchall()
-            selectedRestaurant = value[0]
-            self.primaryId  =  selectedRestaurant[0]
-            self.name =  selectedRestaurant[1]
-            self.address =  selectedRestaurant[2]
-            self.contactName =  selectedRestaurant[4]
-            self.creatorId =  selectedRestaurant[3]
-            self.score =  selectedRestaurant[5]
-            self.profilePicture =  selectedRestaurant[6]
-            self.hours =  selectedRestaurant[7]
-            self.currentStatus =  selectedRestaurant[8]
-
+            if not value:
+                print(value)
```

Comment:
is print necessary?

Reply:
I don't remember that i deleted it @sadikekin  This is your class it this change okay ?

Reply:
It could be relative with my branch , branched before your merge . You can change according  to the appropriate implementation.

Reply:
Thank you for warning i deleted it now.